thrift/contentatom.thrift

```diff
@@ -0,0 +1,5 @@
+include "atoms/tenfour.thrift"
+
+union ContentAtom {
+  1: tenfour.Content tenfour
```

Comment:
Yes, it is a field name of the ContentAtom. Essentially the union says only one field will be present at any one time, and you will need to know which type you are dealing with in order to know which field to look at, or you might be able to just test which field is available, depending on the language you are working in.

So, a `ContentAtom` object will contain, wrapped within it, a payload for a contentatom, but the actual model for that contentatom will vary depending on what type of contentatom it is. So if the type of the contentatom is `tenfour` then you will examine the `tenfour` field to extract the payload, and you can be confident that it will match the `tenfour` model.